content/reference/bindle/provider-bindles.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "Creating a Bindle for a Provider"
+date: 2021-12-29T11:02:05+06:00
+weight: 3
+draft: false
+---
+
+Currently, there is no tooling outside of the `bindle` CLI for creating a bindle for your provider.
+This will be added to `wash` in the future as we solidify our Bindle support. These instructions
+show how to manually assemble a bindle for a provider.
+
+## Prerequisites
+
+Before doing these steps, you'll need to install the `bindle` CLI by downloading it from the
+[releases page](https://github.com/deislabs/bindle/releases).
+
+After downloading `bindle`, set the `BINDLE_URL` environment variable to the bindle server of your
+choosing:
+
+```console
+$ export BINDLE_URL=http://localhost:8080/v1
+```
+
+## Creating your Invoice
+
+First off, you'll need to create an invoice. Create a new file called `invoice.toml` and put the
+following content in it, switching out `name` and `version` with the values for your specific
+project as well as updating the `description` and `authors` field:
+
+```toml
+bindleVersion = "1.0.0"
+
+[bindle]
+name = "wasmcloud.dev/httpserver"
+version = "0.14.7"
+description = "A wasmCloud maintained HTTP server capability provider that implements the wasmcloud:httpserver contract"
+authors = ["The wasmCloud Team"]
+
+[annotations]
+"wasmcloud.dev/type" = "provider"
+```
+
+For more information on Bindle naming and IDs, see the relevant information in the [Bindle
+docs](https://github.com/deislabs/bindle/blob/main/docs/reference-spec.md#bindle-name-and-version).
+Currently, we do not actually use the `wasmcloud.dev/type` annotation, but we likely will in the
+future.
+
+## Adding Parcels
+
+A Provider bindle must contain the `claims.jwt` for a provider as well as all of the binaries needed
+for that provider (generally one per supported OS + ARCH combination). It is recommended that you
+put all of these files in a directory for ease of use when generating the
+[labels](https://github.com/deislabs/bindle/blob/main/docs/label-spec.md). You can then run the
+following command to get all of the necessary labels:
+
+```console
+$ ls subdir/ | xargs -I {} bindle generate-label subdir/{}
```

Comment:
one comment / nit: this shows up with a "copy" button in the docs that will copy this entire block of text. If this is a copyable command, it might be better to separate the command and output

Reply:
Can I turn off the "copy" button here?

Reply:
I think it's default for `bash` / `shell` / `console` so perhaps just "```text" will do it